packages/forma-36-website/src/pages/foundation/typography/design.mdx

```diff
@@ -0,0 +1,128 @@
+import {
+  Card,
+  Typography,
+  Paragraph,
+  Heading,
+  Subheading,
+  SectionHeading,
+  DisplayText
+} from '@contentful/forma-36-react-components';
+
+
+Typography is much more than choosing pretty fonts, it’s a very important components in UI design, Good typography establishes a strong, cohesive visual hierarchy that helps to present content clearly and efficiently to users.
+
+<br />
+
+## Typeface
```

Comment:
Forma 36 enables users to create clean, efficient user interfaces which utilize system fonts.

System font family: